update-vendor.sh

```diff
@@ -44,7 +44,7 @@ clone git github.com/codegangsta/cli 1.1.0
 clone git github.com/coreos/go-systemd v2
 clone git github.com/godbus/dbus v2
 clone git github.com/Sirupsen/logrus v0.7.3
-clone git github.com/syndtr/gocapability 8e4cdcb
+clone git github.com/syndtr/gocapability 66ef2aa
```

Comment:
Adding a pointer to https://github.com/docker/libcontainer/commit/c0640ae4610a32272e53e3964ca2699c89b04f04 for posterity :)